os/drivers/input/ist415.c

```diff
@@ -1076,7 +1103,7 @@ int ist415_initialize(const char *path, struct i2c_dev_s *i2c, struct ist415_con
 	dev->enable = false;
 	dev->pre_enable = false;
 	dev->suspend = false;
-	dev->log = IST415_LOG_LEVEL_ERRO;
+	dev->log = IST415_LOG_LEVEL_INFO;
```

Comment:
We can't set the log level to default info because this may cause performance degradation.